docs/CHANGELOG.rst

```diff
@@ -1,235 +0,0 @@
-
```

Comment:
This file is automatically modified so shouldn't be part of the PR.

Reply:
remove?

Reply:
Yes, please.